Modern languages subject page (#4141)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/explore-subjects/languages.md

---
title: Become a languages teacher
title_paragraph: |-
  <p>
  Learning a language teaches pupils more than just grammar and vocabulary. As a languages teacher, you'll help pupils gain a greater appreciation and understanding of the world.</p>

>   <p>

Reviewer: Still need to change this header @Sarah-DfE - we can't list figures explicitly like this and the line below without a caveat whilst we are in this interim period

Suggest changing to `Tax-free bursaries are available for eligible trainee language teachers.` as Emma has done on other subject pages

(I appreciate this doesn't mention scholarships but that might be the easiest thing for now - they are referenced further down the page.)

>   By teaching pupils languages, you can help them gain a greater appreciation and understanding of the world. As they learn a new language, your pupils will get a unique insight into the countries that speak these languages.</p>

Reviewer 2: ```suggestion
Learning a language teaches pupils more than just grammar and vocabulary. As a languages teacher, you'll help pupils gain a greater appreciation and understanding of the world.</p>
```
```suggestion
  By teaching pupils languages, you can help them gain a greater appreciation and understanding of the world. As they learn a new language, your pupils will get a unique insight into the countries that speak these languages.</p>
```

>   Bursaries of $bursaries_postgraduate_languagesother$ are available for all trainee language teachers. These bursaries include ancient languages.</p> 

Reviewer: We need to dial this back for now as per the holding message PR - we can't say amounts without a caveat so @emmaachesongray has made the other subject pages very generic in this header section - need to do the same here!

>   Bursaries of $bursaries_postgraduate_languagesother$ are available for all trainee language teachers. These bursaries include ancient languages.</p> 

Reviewer 2: ```suggestion
Tax-free bursaries are available for all trainee language teachers. These bursaries include ancient languages.</p> 
```

  <p>
  If you're training to teach French, German or Spanish, then you could be eligible for a scholarship of $scholarships_languagesfrenchgermanspanish$.</p>
description: |-

>    When you teach languages, you'll help pupils develop their cultural awareness and communication skills. Find out what you'll teach and what funding is available to help you train.

Reviewer: Is this slightly long re SEO best practice?

>    When you teach languages, you'll help pupils develop their cultural awareness and communication skills. Find out what you'll teach and what funding is available to help you train.

Reviewer 2: ```suggestion
Teaching languages helps pupils develop their cultural awareness and communication skills. Explore what you'll teach and what funding is available for training.
```
